lib/jam/addLine.ts

```
@@ -0,0 +1,223 @@
+    if (end) {
+      path.path[1][3] = end.x;
+      path.path[1][4] = end.y;
+    }
+
+    updateCurvature(path);
+    path.setCoords();
+
+    // Force control points to update their positions
+    if (path.controls) {
+      Object.values(path.controls).forEach((control) => {
+        if (control && control.positionHandler) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)